### PR TITLE
features: add gate-auto-refresh-hook feature flag

### PR DESCRIPTION
**Pull request by stolowski**
Introduce refresh-control feature flag.

### CODE REVIEW

**Review by anonymouse64 — APPROVED** [review]
lgtm

File: features/features.go [hunk 1]
```diff
@@ -94,6 +96,8 @@ var featureNames = map[SnapdFeature]string{
 	CheckDiskSpaceInstall: "check-disk-space-install",
 	CheckDiskSpaceRefresh: "check-disk-space-refresh",
 	CheckDiskSpaceRemove:  "check-disk-space-remove",
+
+	RefreshControl: "refresh-control",
```
Comment (pedronis) [inline features/features.go:100]: as I said elsewhere we used this name already for something else (the original refresh control via validation assertion); so we need to think a name for this

**Review by pedronis — CHANGES_REQUESTED** [review]
we need to think a different or more precise name

File: features/features.go (hunk 1 above)
Comment (pedronis) [reply features/features.go:100]: the easiest is maybe to call it based on the hook:  "gate-auto-refresh-hook" ?

Comment (stolowski) [reply features/features.go:100]: Yes, that makes sense, +1

Comment (stolowski) [reply features/features.go]: Done.

**Review by pedronis — APPROVED** [review]
thank you

**Review by anonymouse64 — APPROVED** [review]

**Pull request merged**